packages/connect-iframe/package.json

```diff
@@ -3,7 +3,9 @@
     "version": "9.0.0",
     "private": true,
     "scripts": {
-        "build:iframe": "TS_NODE_PROJECT=\"tsconfig.json\" yarn webpack --config ./webpack/prod.webpack.config.ts --stats-children",
+        "lint:js": "yarn g:eslint '**/*.{ts,tsx,js}'",
```

Comment:
is this line relevant? maybe it went away as the result of refactorings by @peter-sanderson ?

Reply:
Ups :upside_down_face:  that was definitely not intended result of some rebase. Fixed it here https://github.com/trezor/trezor-suite/pull/14959/commits/5d30b2460f81f244b678f840a9841e577f9a1e7f